src/partners.html

```diff
@@ -84,26 +84,29 @@ <h2 class="my-md-4 my-3">California Department of Technology</h2>
         <div class="row justify-content-md-center align-content-stretch my-md-4 my-3">
           <div class="col-12 col-md-12 col-lg-6">
             <div class="background-purple-4 py-4 py-md-5 my-md-2 px-3 text-center clipped-stat-start">
-              <h2 class="fs-stats text-white pt-4 pt-md-3">68%</h1>
-              <h3 class="text-white d-block mb-3 mb-md-0">Increase in reduced-fare transactions<sup id="fnref:1" role="doc-noteref"><a class="text-decoration-none text-white" href="#fn:1" rel="footnote">1</a></sup></h3>
+              <h2 class="fs-stats text-white pt-4 pt-md-3">
+                68%
+              </h1>
```

Comment:
I just noticed that this closing tag doesn't match the opening :eyes: Obviously this wasn't introduced in this PR, but thought I'd note it. We can fix in a separate PR